File: app/src/main/java/com/wwxiaoqi/wechat_backup/utils/AppUtils.java
package com.wwxiaoqi.wechat_backup.utils;

import android.content.Context;
import android.content.SharedPreferences;

public class AppUtils {

  public static boolean isFirstRunApp(Context context) {
    SharedPreferences sharedPreferences = context.getSharedPreferences("share", Context.MODE_PRIVATE);
    boolean isFirstRun = sharedPreferences.getBoolean("isFirstRun", true);
    SharedPreferences.Editor editor = sharedPreferences.edit();
    if (isFirstRun) {
      editor.putBoolean("isFirstRun", false);
      editor.apply();
      return true;
    }
    return false;
  }

}

File: app/src/main/java/com/wwxiaoqi/wechat_backup/utils/AppUtils.kt
package com.wwxiaoqi.wechat_backup.utils

import android.content.Context

object AppUtils {

  /**
   * 判断 app 是否第一次运行
   * @param context Context
   */
  fun isFirstRunApp(context: Context): Boolean {
    val sharedPreferences = context.getSharedPreferences("share", Context.MODE_PRIVATE)
    val isFirstRun = sharedPreferences.getBoolean("isFirstRun", true)
    val editor = sharedPreferences.edit()
    if (isFirstRun) {
      editor.putBoolean("isFirstRun", false)
      editor.apply()
      return true
    }
    return false
  }
}